cpp/ql/test/library-tests/types/datasizeof/datasizeof.ql

```diff
@@ -0,0 +1,10 @@
+import cpp
+
+from DatasizeofOperator sto, string elemDesc, Element e
```

Comment:
Question: Would it be helpful to also output the constant value of the expression?

Reply:
Added. Also for the `sizeof` test from which this was copied.